migrations/sqls/20190806211241-client-interaction-up.sql

```diff
@@ -0,0 +1,33 @@
+CREATE TABLE IF NOT EXISTS clientInteractions (
+    id INT AUTO_INCREMENT PRIMARY KEY,
+    clientInteractionType ENUM (
+        'clientInteractionCreated',
+        'clientInteractionUpdated',
+        'clientInteractionDeleted'
+    ) NOT NULL,
+    clientId INT NOT NULL,
+    title VARCHAR(255) NOT NULL,
```

Comment:
This gets tricky -- I'm not sure exactly what to do here. We already have `title` and `description` columns [here](https://github.com/JustUtahCoders/comunidades-unidas-internal/blob/master/migrations/sqls/20190629230418-activity-log-up.sql#L4). My idea for client interactions is that the data will be split up between the clientLogs table and the clientInteractions table. Anything that is in the clientLogs table will show up in the Client "History" tab automatically, so it's desireable to put information there instead of in clientInteractions.

Regarding the title -- for many of the clientLogs, we just put in a standard title for the logType. For example, see [this line of code](https://github.com/JustUtahCoders/comunidades-unidas-internal/blob/9c8ec0fff60ef58fa220283e5abb9c4b925c14c4/backend/apis/clients/insert-client.utils.js#L30) where the title is always "Contact information was updated."

The only exception to this has been for case notes where we allow the user to type in a title for the case note.

So for client interactions we have the choice of either letting the user specify a title or just creating a default title. I actually lean towards using a default title of "Client interaction added for service SNAP".

Your thoughts?

Reply:
I think the list would work out the best.

Reply:
What list are you referring to?

Reply:
A* list, sorry. Of default titles.